packages/coqide/coqide.8.7.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "1.2"
+maintainer: "Ralf Jung <post@ralfj.de>"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "https://github.com/coq/coq.git"
+license: "LGPL-2.1"
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+    "-usecamlp5"
```

Comment:
Note that the `-usecamlp5` option is deprecated, since camlp5 is already a required dependency.

Reply:
Good point, the option is not even listed any more in `./configure -help`. I removed it.